src/policy.c

```diff
@@ -191,26 +191,23 @@ qd_error_t qd_register_policy_manager(qd_policy_t *policy, void *policy_manager)
 }
 
 
-long qd_policy_c_counts_alloc()
+void *qd_policy_c_counts_alloc()
 {
-    qd_policy_denial_counts_t * dc = NEW(qd_policy_denial_counts_t);
+    qd_policy_denial_counts_t *dc = NEW(qd_policy_denial_counts_t);
```

Comment:
What's the convention regarding that space, actually?

Reply:
Good question! C type syntax is a mess.  I don't think we've formally established a convention here, but personally I put the "star" right next to the var name. I do this because "star" binds right-to-left, which means it binds with the var, not the type.  So if you wrote this "c" code:

int* x, y; 

it seems like the code is defining two pointers to int, but in fact the * binds with the x, so you actually get a pointer to int (x) and an int (y).

Doing it this way:

int *x, y;

makes it clearer that x is the pointer.  Not that you'd actually write that code, but it is legal C.

Reply:
`clang-format` all the things!